### ISSUE TITLE
non-unique id is used for the link in quoteblock

### ISSUE THREAD

**Issue by lwchkg** [azu/gitbook-plugin-include-codeblock#28]
Here's some generated code in the generated code. Note `<a id="test1.vb">`, where test1.vb can be imported more than once.

``` html
<div class="quoteblock">
<blockquote>
<div class="paragraph">
<p class="comments-section"><a id="test1.vb"></a><a href="test1.vb">test1.vb</a><div class="comments-icon"><div class="marker">+</div></div></p>
</div>
</blockquote>
</div>
```

**Comment by lwchkg** [comment]
Tested in https://lwchkg.gitbooks.io/testing/content/asciidoc_testing.html
At least the second id attributes should be changed. The second id should be anything else but not "test1.vb" (e.g. "test1.vb[1]" is okay).

Anyway, if you like a pulling I can write it for you.

**Comment by azu** [comment]
It seem resolved by template options #30
